Clicker/Quarks/PollQuark.swift

```diff
@@ -176,7 +180,8 @@ struct UpdatePoll: ClickerQuark {
                 throw NeutronError.badResponseData
             }
             let questionType: QuestionType = (type == Identifiers.multipleChoiceIdentifier) ? .multipleChoice : .freeResponse
-            return Poll(id: id, text: text, results: results, type: questionType, isShared: shared)
+            let state: PollState = shared ? .shared : .ended
```

Comment:
I appreciate the use of enums in this file instead of passing the boolean through to the Poll initializer. Nice!